src/CSharpMongoMigrations.Tests/Migrations/MigrationTests.cs

```diff
@@ -1,4 +1,5 @@
-﻿using FluentAssertions;
+﻿//using CSharpMongoMigrations.Migrations;
```

Comment:
Unnecessary commented directive. It should be removed.

Reply:
Removed.